### ISSUE TITLE
Publication Tab Update

### ISSUE THREAD

**Issue by euisuny** [plclub/plclub-web-2#3]
Feedback from @cdstanford:

Rajeev doesn't have a .bib file so that is a problem for making [the publication] list complete.

Maybe the simplest solution will be to include, at the top of this page, links to each professor's individual publications page, like "Benjamin PIerce's publications" "Rajeev Alur's publications" etc. And then we can keep the combined list if we want, but it seems less important. I am not sure if we need to maintain a combined list at all, it seems like it might not be worth the work.

**Comment by euisuny** [comment]
Next on TODO: @cdstanford --- do we want to keep this as a partial list, or should we move towards a completed list, categorized by year? It would be neat if we could inch towards a complete list (maybe we need to nudge Rajeev a little bit).

**Comment by cdstanford** [comment]
A complete list would be nice! Although I am not sure I could convince Rajeev to maintain another file on his website, it looks like DBLP allows exporting a page of publications as bibTeX. Maybe we can do that?

**Comment by cdstanford** [comment]
See https://dblp.uni-trier.de/pers/hb/a/Alur:Rajeev.html

**Comment by cdstanford** [comment]
Probably fine to just store the .bib file in our git and re-download it from DBLP periodically.

Also if Osbert's publications list is still not working properly we can get it from DBLP. We could get all of them from DBLP for that matter.

**Comment by euisuny** [comment]
Oh, that's neat! Yes, that sounds like a smart alternative, if we can't connive Rajeev and/or Osbert to ride the .bib train.

**Comment by dunnl** [comment]
There's already some infrastructure to use Pandoc to do some runtime parsing. So it shouldn't be too hard to parse the HTML into a .bib and feed it in with the rest. I'll work on it.
